app/config/ezplatform.yml

```diff
@@ -37,6 +41,20 @@ ezpublish:
             http_cache:
                 purge_servers: ["%purge_server%"]
             design: main
+        learning_group:
+            content:
+                tree_root:
+                    location_id: 237
+            # Pool to use for cache, needs to be different per repository (database).
+            cache_pool_name: '%cache_pool%'
+            # These reflect the current installers, complete installation before you change them. For changing var_dir
+            # it is recommended to install clean, then change setting before you start adding binary content, otherwise you'll
+            # need to manually modify your database data to reflect this to avoid exceptions.
+            var_dir: var/learning
+            # HttpCache purge server(s) setting, eg Varnish, for when ezpublish.http_cache.purge_type is set to 'http'.
+            http_cache:
+                purge_servers: ["%purge_server%"]
+            design: main
```

Comment:
should they use different designs?

eg:
com: [com, ezplatform]
learn: [learn, ezplatform]

Reply:
You are totally right. We will bring the themes when we will submit the PR for the templates :)